src/pages/AddPatient/AddPatient.js

```diff
@@ -150,185 +150,204 @@ const AddPatient = () => {
         </InfoWrapper>
       </Headings>
       <Container>
-        <Form onSubmit={handleSubmit(handleSave)}>
-          <Row>
-            <Col md={{ size: 6 }}>
-              <InputField
-                title="Cell Phone"
-                name="phone"
-                required
-                error={getErrorMessage(errors, 'phone')}
-                innerRef={register}
-                defaultValue="+91-"
-              />
-            </Col>
-            <Col md={{ size: 6 }}>
-              <InputField
-                title="Contact Email"
-                name="email"
-                type="email"
-                innerRef={register}
-              />
-            </Col>
-          </Row>
-          <Row>
-            <Col md={{ size: 6 }}>
-              <InputField
-                title="First Name"
-                name="firstName"
-                required
-                innerRef={register}
-                error={getErrorMessage(errors, 'firstName')}
-              />
-            </Col>
-            <Col md={{ size: 6 }}>
-              <InputField
-                title="Last Name"
-                name="lastName"
-                innerRef={register}
-              />
-            </Col>
-          </Row>
-          <Row>
-            <Col md={{ size: 3 }}>
-              <FormGroup check row className="mx-0 pl-0 form-group">
-                <Label>Gender</Label>
-                <div className="d-flex mt-3 flex-wrap">
-                  {GENDER_OPTIONS.map(({ label, value }) => (
-                    <RadioLabel
-                      htmlFor={value}
-                      key={value}
-                      onClick={() => setGender(value)}>
-                      <RadioInput
-                        type="radio"
-                        name="gender"
-                        value={value}
-                        id={value}
-                      />
-                      <OptionName checked={checkedGender === value}>
-                        {label}
-                      </OptionName>
-                    </RadioLabel>
-                  ))}
-                </div>
-              </FormGroup>
-            </Col>
-            <Col md={{ size: 4 }}>
-              <Label>Date of Birth</Label>
-              <SplittedDatePicker date={dateOfBirth} setDate={setDateOfBirth} />
-            </Col>
-            <Col md={{ size: 3 }}>
-              <Label>Height</Label>
-              <div className="d-flex">
-                <div className="flex-grow-1 mr-2">
-                  <InputField
-                    type="number"
-                    name="heightFt"
-                    innerRef={register}
-                    customClass="measurement ft pr-3"
-                    error={getErrorMessage(errors, 'heightFt')}
-                    min={0}
-                    max={8}
-                  />
+        {!isConfirmed ? (
+          <Form onSubmit={handleSubmit(handleSave)}>
```

Comment:
it looks like Form is similar in AddPatient and EditPatient: please check if it so and move to a separate component to avoid duplication

Reply:
Though the forms are similar, there are some changes done in EditPatient page form to pre-load the existing values on render. So it needs to be separated